src/Module.Client/GUI/MapSpriteData.xml

```diff
@@ -1103,6 +1104,15 @@
       <SheetY>0</SheetY>
       <CategoryName>ui_mploading</CategoryName>
     </SpritePart>
+    <SpritePart>

```

Comment:
It also needs a GenericSprite at the end of the file

Reply:
done